### PR TITLE
Allow specification of the IP address to check via environment variable

### PR DESCRIPTION
**Pull request by nchelluri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ your proxy frontend),
 [mod_remoteip](http://httpd.apache.org/docs/current/mod/mod_remoteip.html) may
 be used to set the client IP address.
 
+Manually setting the client IP address is also possible - see
```
Comment (oschwald) [inline README.md]: Get rid of the hyphen and add a period. "See" is the start of a new sentence.

File: README.md [hunk 2]
```diff
@@ -120,6 +123,17 @@ In addition to the environment variable specified by `MaxMindDBEnv`, this
 module exports `MMDB_ADDR`, which contains the IP address used for lookups by
 the module. This is primarily intended for debugging purposes.
 
+## Client IP address lookup control ##
+
+In case you want supply your own value for the IP address to lookup, it may be
+done by setting the environment variable `MMDB_ADDR`.
+This can be done, for instance, with
+[ModSecurity](https://github.com/SpiderLabs/ModSecurity/) in (real) phase 1.
```
Comment (oschwald) [inline README.md]: Should we note that this doesn't actually work with ModSecurity out of the box (at least on most distros) and that you need to specially compile it? My primary objection to this change is that it is almost impossible to use this correctly.

File: t/conf/extra.conf.in [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+LoadModule maxminddb_module     /usr/lib/apache2/modules/mod_maxminddb.so
+
+LoadModule unique_id_module     /usr/lib/apache2/modules/mod_unique_id.so
+LoadModule security2_module     /usr/lib/apache2/modules/mod_security2.so
+
 ScriptSock /tmp/cgisock
 ScriptAlias /cgi-bin/ "@ServerRoot@/cgi-bin/"
 
-LogLevel        debug
+#LogLevel        debug
```
Comment (oschwald) [inline t/conf/extra.conf.in]: We shouldn't leave uncommented code. Maybe we could switch it back to `debug` unless there is a good reason not to.

**Comment by nchelluri** [comment]
Closing this PR, updated code is in #66

**Pull request closed without merging**